### ISSUE TITLE
x360ce + Original NFS Rivals

### ISSUE THREAD

**Issue by snap171** [x360ce/x360ce#136]
Hi,

I bought NFS Rivals on Origin. I have Logitech Dual Action. I would like to use my PAD in this game but I can't.
1. The newest x64 x360ce with administrator privileges returns error when starting.
2. The newest x86 x360ce with administrator privileges run, I can't configure it etc.
3. Still the right, upper trigger is A button, and right, lower trigger is B button. So program actually don't work. 
4. The newest x64 x360ce with administrator privileges returns error when starting.
5. The newest x86 x360ce with administrator privileges run, I can't configure it etc.
6. Still I the right, upper trigger is A button, and right, lower trigger is B button. So program actually don't work. 
7. x86 x360ce recognizes controller correctly and the button binding works. But after running the game it does not work.

To summarize: will this program work with original game?

**Comment by Squall-Leonhart** [comment]
What error?  you haven't provided anything relevant that would assist us in helping you.
